CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Getting Started with Contributing
+
+The first thing you need to do is find an issue to work on. Go to Issues and find a suitable issue which you think you can handle and are willing to work on. Make sure you ask the maintainers to have the issue assigned to you before you start working on it.
+
+
+**Step 1:** Fork the repository
+
+**Step 2:** Clone the forked repository to your local machine with `git clone <repository link>`
+
+**Step 3:** Add a remote (upstream) to your fork using: 
+```
+git remote add upstream https://github.com/postmanlabs/newman-dashboard.git
+```
+
+**Step 4:** Synchronize your fork with upstream using the following commands:
+
+```
+
+$ git checkout develop
+
+$ git fetch upstream 
+
+$ git merge upstream/develop
+
+$ git push origin develop
+
+```
+
+**Step 5:** Make a branch with a suitable name relevant to the issue you are working on using `git branch <branch>`
+
+**Step 6:** Switch your working branch from master to the newly created branch with `git checkout <branch>`
+
+**Step 7:** Work on the changes. You can keep track of your branch and staging area using `git status`
+
+**Step 8:** Add all of the changes you made to the staging area before you commit with `git add .` 
+
+**Step 9:** Commit your changes with a meaningful commit message using `git commit -m "commit message"`
+
+**Step 10:** Finally push your changes to your forked repository using `git push origin <branch>`
+
+
+
+
+## Usage
+
+This package is both a reporter and a standalone package. The standalone package takes care of launching the dashboard, while the reporter connects to the newman run.
+
+A. For installation:
+```
+npm pack reporter
+```
+
+```
+npm i -g newman-reporter-dashboard-0.0.1.tgz
+```
+
+B. Launch the dashboard on `localhost:5001`

```

Comment:
We should also add the steps to start the dashboard in dev mode:
```
npm run dev:dashboard
```

Reply:
Before launching the dashboard, the frontend needs to be built. This can be done in two ways:
1.
```
npm run build
```
or
2.
```
cd frontend
npm run build
```

Reply:
cc @the-rdt this might be why you're getting an error? 🤔